doc/config_options.txt

```diff
@@ -778,6 +778,12 @@ Specify either a cron expression (`<minute> <hour> <dom> <month> <dow>`), a comm
 
 <!-- config group instance-snapshots end -->
 <!-- config group instance-volatile start -->
+```{config:option} volatile.<name>.<target_path>.dir_created_by_lxd instance-volatile
```

Comment:
Do we required `name` and `target_path`? Or could the device name alone be sufficient? If you name the device `mnt` and mount it to `mnt` inside the container you get `volatile.mnt.mnt.dir_created_by_lxd` which might be redundant.

Reply:
Yes, I can try to chhange the logic a bit to remove `target_path` from the key.

Reply:
fixed